i18n/en/docusaurus-plugin-content-docs/current/08-数据集成/29RocketMQ Connect In Action2.md

```diff
@@ -1,53 +1,56 @@
-# RocketMQ Connect实战2
+# RocketMQ Connect actual combat 2
```

Comment:
RocketMQ Connect in Action 2

Reply:
@tsunghanjacktsai  I will fix it and re-submit later